source/content/wordpress-cron.md

```diff
@@ -71,62 +86,79 @@ terminus wp <site>.<env> -- cron <your wp-cron command and switches here>
+
+Terminus cannot execute Cron Jobs if you enable the Security setting on your Dashboard. You may see status report errors on the Dashboard as a result.
+
 </Alert>
 
 ### Plugins
 
-If you want to keep an eye on WP-Cron but don't like the command line, there are several plugins you can use. [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/screenshots/ "WP Crontrol page on wordpress.org"), for example, will show you all of the events scheduled for your site. You can create, edit, run, or delete jobs immediately from within your WordPress admin dashboard. You can also hook new actions into schedules or move existing actions to new schedules from within the Tools section.
+There are several plugins you can use if you want to keep an eye on WP-Cron but don't like the command line. [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/screenshots/ "WP Crontrol page on wordpress.org"), for example, will show you all of the events scheduled for your site. You can create, edit, run, or delete jobs immediately from within your WordPress admin dashboard. You can also hook new actions into schedules or move existing actions to new schedules from within the Tools section.
 
-[WPX Cron Manager Lite](https://wordpress.org/plugins/wpx-cron-manager-light/ "WPX Cron Manager Lite") will do a similar job with a slightly different UI. This plugin requires you to do a one-time installation of the WPX framework, which you can do straight from the plugin manager page.
+[WPX Cron Manager Lite](https://wordpress.org/plugins/wpx-cron-manager-light/ "WPX Cron Manager Lite") works similarly with a slightly different UI. This plugin requires you to do a one-time installation of the WPX framework, which you can do straight from the plugin manager page.
 
 ## Manage WP-Cron Externally
 
-If you're looking for more control over your site's cron jobs, or you don't want WP-Cron to handle tasks internally, you can use external crons instead. This will solve the problems discussed above for high traffic and low traffic sites.
+You can use external Crons if you want more control over your site's Cron jobs, or if you don't want WP-Cron to handle jobs internally. This will solve the problems with high traffic and low traffic sites discussed in the Troubleshooting section.
 
 ### Disable WP-Cron
 
-The first thing you'll need to do is disable WP-Cron's internal processing. Add the following line to your `wp-config.php` file:
+Disable WP-Cron's internal processing by adding following code to your `wp-config.php` file:
 
 ```php:title=wp-config.php
 define('DISABLE_WP_CRON', true);
 ```
 
 <Alert title="Note" type="info">
-There are many important tasks that WP-Cron takes care of, so be prepared to complete all the steps below.
+
+WP-Cron executes many important jobs. Be sure to complete all the steps below.
```

Comment:
This note doesn't seem necessary now that we've added Pantheon Cron to the platform.